### PR TITLE
Adding instructions about dense nodes (BSC#1177776)

### PR DESCRIPTION
**Pull request by asettle**
Requires release note

### CODE REVIEW

**Comment by asettle** [comment]
@sknorr this change will require an addition to the release nodes. This note should say (something to the effect of): _SUSE Enterprise Storage cannot deploy more than 62 OSDs per node. If you intend to update a cluster with very dense nodes (>62 OSDs per node) read <link to doc section> before proceeding._ 

I believe putting it under Known ISsues would be suitable. All okay with you @jan--f ?

**Comment by jan--f** [comment]
> @sknorr this change will require an addition to the release nodes. This note should say (something to the effect of): _SUSE Enterprise Storage cannot deploy more than 62 OSDs per node. If you intend to update a cluster with very dense nodes (>62 OSDs per node) read `link to doc section` before proceeding._
> 
> I believe putting it under Known ISsues would be suitable. All okay with you @jan--f ?

I'd like to propose a slight change to the release notes entry:
```
SUSE Enterprise Storage cannot deploy more than 62 OSDs per node in its default configuration. If you intend to update a cluster with very dense nodes (>62 OSDs per node) read `link to doc section` before proceeding.
```
_Known Issues_ section sounds alright.

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -1193,6 +1193,17 @@ config: OK
      deployment will use the intended cluster network from the start.
     </para>
    </note>
+   <para>
+     If your cluster is set to have dense nodes (greater than 60 OSDs per host),
```
Comment (jan--f) [inline xml/deploy_cephadm.xml:1197]:
```suggestion
     If your cluster is set to have dense nodes (greater than 62 OSDs per host),
```
Just for consistency its probably better to use the same number? Apologies, should have thought of that in the initial proposal.

**Review by jan--f — APPROVED** [review]
One minor nit, but otherwise LGTM!

**Review by jan--f — APPROVED** [review]
:+1:

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -1193,6 +1193,21 @@ config: OK
      deployment will use the intended cluster network from the start.
     </para>
    </note>
+   <para>
+     If your cluster is set to have dense nodes (greater than 62 OSDs per host),
+     make sure to assign sufficient ports for &ceph; OSDs. The default (6800-7300)
+     currently allows for no more than 62 OSDs per host. For a cluster with dense
+     nodes, adjust the setting <literal>ms_bind_port_max</literal> to a
+     suitable value. Each OSD will consume 8 additional ports. For example,
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1201]:
```suggestion
     suitable value. Each OSD will consume eight additional ports. For example,
```

File: xml/deploy_cephadm.xml [hunk 3]
```diff
@@ -1193,6 +1193,21 @@ config: OK
      deployment will use the intended cluster network from the start.
     </para>
    </note>
+   <para>
+     If your cluster is set to have dense nodes (greater than 62 OSDs per host),
+     make sure to assign sufficient ports for &ceph; OSDs. The default (6800-7300)
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1198]: I am not sure it is, but if this is a _range_ of ports, then I think "the default range (6800-7300)" would be preferable.

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/deploy_cephadm.xml (hunk 3 above)
Comment (asettle) [reply xml/deploy_cephadm.xml:1198]: Good question, I presume it's a range based on how we've phrased this here: https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#storage-bp-net-firewall

@jan--f ?

Comment (jan--f) [reply xml/deploy_cephadm.xml:1198]: Yep makes sense!

Comment (asettle) [inline xml/deploy_cephadm.xml:1198]:
```suggestion
     make sure to assign sufficient ports for &ceph; OSDs. The default range (6800-7300)

**Review by lproven — APPROVED** [review]

**Pull request merged**